src/index.ts

```diff
@@ -176,7 +176,7 @@ export default class HorizontalScroll extends EventEmitter {
         }
         const distance = Math.max(this.container.scrollLeft + offset, 0);
 
-        if (distance < this.container.scrollWidth - this.container.clientWidth) {
+        if (distance - SCROLL_AMOUNT < this.container.scrollWidth - this.container.clientWidth) {
```

Comment:
Hey, thanks for contributing! Works fine, but this should be changed to the `scrollAmount` that's passed to the constructor so it takes the actual scrollAmount that's configured. Which is by default equal to the `SCROLL_AMOUNT` variable, but this makes it more dynamic.

Reply:
Resolved in PR #32